### PR TITLE
test/integration: fix already used port race

### PR DESCRIPTION
**Pull request by squat**
Occasionally, our integration tests are flaking because a process
listening on a certain port in the first integration test has not exited
before another process tries to listen on that port in the second
integration test. This commit gives the processes in the second
integration test a separate set of ports to ensure that the integration
tests do not conflict with one-another.

Signed-off-by: Lucas Servén Marín <lserven@gmail.com>

cc @metalmatze

### CODE REVIEW

File: test/integration-v2.sh [hunk 1]
```diff
@@ -9,41 +9,41 @@ set -euo pipefail
 result=1
```
Comment (bwplotka) [inline test/integration-v2.sh]: I would at some point rewrite it to https://github.com/thanos-io/thanos/blob/master/test/e2e/e2ethanos/services.go#L154

Comment (squat) [reply test/integration-v2.sh]: yeah, I've been thinking about this for a while; this repo has a lot of space for refactoring, both the tests and the actual packages. These recent changes will at least unblock us to keep cleaning up

**Pull request merged**